src/main/java/com/iexec/worker/tee/post/compute/worker/WorkerApiManager.java

```diff
@@ -16,11 +16,14 @@
 
 package com.iexec.worker.tee.post.compute.worker;
 
+import com.iexec.common.utils.EnvUtils;
 import com.iexec.common.utils.FeignBuilder;
 import feign.Logger;
+import org.apache.commons.lang3.StringUtils;
 
 public class WorkerApiManager {
-    private static final String WORKER_HOST = "worker:13100";
+    public static final String WORKER_HOST_ENV_VAR = "WORKER_HOST";
```

Comment:
private :)

Reply:
This was required with the previous version, but no more. Thanks for the notice! Please see https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/71/commits/2e8e6c2102cb8ca2f731c4b50647bf1e067e4b47.